metrics/grafana/tiflash_summary.json

```diff
@@ -10677,7 +10677,7 @@
           "targets": [
             {
               "exemplar": true,
-              "expr": "sum(delta(tiflash_raft_raft_log_lag_count_bucket{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", type=~\"applied_index\"}[1m])) by (le, type)",
```

Comment:
Could you also help me to change this metrics into `tsbuckets`?
It will be like this
<img width="1009" alt="image" src="https://github.com/pingcap/tiflash/assets/5670241/b825bb04-29d4-498b-9755-8466de7b4d5c">

Reply:
Fixed